storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -1835,6 +1835,21 @@
     public static final String TOPOLOGY_CLASSPATH="topology.classpath";
 
     /**
+     * Topology-specific classpath for the worker child process. This will be *prepended* to
+     * the usual classpath, meaning it can override the Storm classpath. This is for debugging
+     * purposes, and is disabled by default. To allow topologies to be submitted with user-first
+     * classpaths, set the user.classpath.first.enabled config to true.
+     */
+    @isStringOrStringList
+    public static final String TOPOLOGY_CLASSPATH_FIRST="topology.classpath.first";
+
+    /**
+     * Enables user-first classpath. See topology.classpath.first
+     */
+    @isBoolean
+    public static final String STORM_USER_CLASSPATH_FIRST_ENABLED="storm.user.classpath.first.enabled";
```

Comment:
This, unlike the above, should not be in the `topology.*` namespace, because `topology.*` configs are specific per-topology by convention.
Maybe `storm.topology.classpath.prefix.enabled`?